rosdep/base.yaml

```diff
@@ -2814,6 +2814,13 @@ libgphoto-dev:
     precise: [libgphoto2-2-dev]
     raring: [libgphoto2-2-dev]
     saucy: [libgphoto2-6-dev]
+libgpiod-dev:
+  arch: [libgpiod]
```

Comment:
Packages in AUR are not allowed - please see the [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#arch-linux) for details.
```suggestion
```

Reply:
oops. Good catch.